.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'Bug'
+assignees: '@timstackblock'
```

Comment:
Is Tim the right person to assign here? Just curious to see what your thought process is. I assume you'd like the Testing team to validate first?

Reply:
I'll remove this as well so it gets triaged as normal.